.travis.yml

```diff
@@ -25,6 +25,7 @@ script:
 - yarn build
 - yarn test
 - yarn test:size
+- yarn release
```

Comment:
It checks the following things to see if it can release it:

- see if the latest commit message starts with `chore: release v${currentVersion}`
- see if the current branch is `master` or `maint`.
  - If it's `master`, then it's coming from `develop`. It will release this, and merge it back to `develop`.
  - If it's `maint`, then it's coming from `maint`. It will release this.

If the conditions are met, then it releases.